drivers/mtd_mapper/mtd_mapper.c

```diff
@@ -78,6 +78,7 @@ static int _init(mtd_dev_t *mtd)
     assert(backing_mtd->page_size == region->mtd.page_size);
     assert(backing_mtd->pages_per_sector == region->mtd.pages_per_sector);
     assert(backing_mtd->sector_count >= region->mtd.sector_count);
+    assert(backing_mtd->write_size == region->mtd.write_size);
```

Comment:
Can't we instead just

```suggestion
    backing_mtd->write_size = region->mtd.write_size;
```

Seems pretty redundant to require the user to set this.

Reply:
We could, but I value consistency more highly here (page size and pages per sector are set manually as well).